### PR TITLE
Project 1: create animals table.

### PR DESCRIPTION
**Pull request by AbdimulhinYussuf3675**
- [x] Create a database named `vet_clinic`.
- [x] Create a table named animals with the following columns [(schema.sql)](https://github.com/AbdimulhinYussuf3675/Vet-clinic/blob/animals_table/schema.sql)
       `id`: integer
       `name`: string
       `date_of_birth`: date
       `escape_attempts`: integer
       `neutered`: boolean
       `weight_kg`: decimal
- [x] Insert test data into the animals table [(data.sql)](https://github.com/AbdimulhinYussuf3675/Vet-clinic/blob/animals_table/data.sql)
- [x] Write queries for the following quests [(queries.sql)](https://github.com/AbdimulhinYussuf3675/Vet-clinic/blob/animals_table/queries.sql):

-       Find all animals whose name ends in "mon".
Screenshot from 2023-03-28 16-02-18
-       List the name of all animals born between 2016 and 2019.
Screenshot from 2023-03-28 16-03-43
-       List the name of all animals that are neutered and have less than 3 escape attempts.
Screenshot from 2023-03-28 16-04-38
-       List the date of birth of all animals named either "Agumon" or "Pikachu".
Screenshot from 2023-03-28 16-05-18
- List name and escape attempts of animals that weigh more than 10.5kg
Screenshot from 2023-03-28 16-05-52
-       Find all animals that are neutered.
Screenshot from 2023-03-28 16-06-25
-       Find all animals not named Gabumon.
Screenshot from 2023-03-28 16-06-56
-       Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
Screenshot from 2023-03-28 16-08-19

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+CREATE TABLE IF NOT EXISTS animals (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(30) NOT NULL,
+    date_of_birth DATE,
+    escape_attempts NUMERIC(5,2),
+    neutered BOOLEAN NOT NULL,
+    weight_kg DECIMAL NOT NULL);
```
Comment (Olamarx) [inline queries.sql]: - [ ] This should be in the schema.sql file as it is creating and not querying.

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name like '%mon';
+
+SELECT name FROM animals WHERE DATE_PART('Year', date_of_birth) BETWEEN 2016 AND 2019;
+
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
```
Comment (Olamarx) [inline schema.sql]: - [ ] These are queries and they should be in the queries.sql file.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @AbdimulhinYussuf3675,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Readme file ✔️ 
- Queries made ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @AbdimulhinYussuf3675, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Great job addressing the comment from the previous code reviewer :man_technologist: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**